tools/jbrowse2/jbrowse2.xml

```diff
@@ -0,0 +1,873 @@
+<tool id="jbrowse2" name="JBrowse2" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.05">
+    <description>genome browser</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <xrefs>
+        <xref type="bio.tools">jbrowse2</xref>
+    </xrefs>
+    <expand macro="requirements"/>
+    <expand macro="creators"/>
+    <version_command>python '${__tool_directory__}/jbrowse2.py' --version</version_command>
+    <command detect_errors="aggressive"><![CDATA[
+mkdir -p '$output.files_path' &&
+cp '$trackxml' '$output.files_path/galaxy.xml' &&
+export JBROWSE2_PATH=\$(dirname \$(which jbrowse))/../opt/jbrowse2 &&
+#if $jbgen.ucol.formcoll=="collect":
+    python '$__tool_directory__/autogenJB2.py'
+    #for $key in $autoCollection.keys():
+        #if $autoCollection[$key].is_collection:
+            #set subCol=$autoCollection[$key]
+            #set pafs=[($subCol[x],$subcol[x].ext,x) for x in $subCol.keys() if $subCol[x].ext == 'paf']
+            #if len($pafs) > 0:
+                --pafmeta '$pafs[0]'
+                #set refs = [($pafs[0][2],$subCol[x],x) for x in $subCol.keys() if $subCol[x].ext == 'fasta']
+                #for $ref in $refs:
+                    --pafreferencemeta '$ref'
+                #end for
+            #end if
+        #else if $autoCollection[$key].ext == 'fasta':
+            --referencemeta '$autoCollection[$key],$autoCollection[$key].ext,$key'
+        #else if $autoCollection[$key].ext in ['bed', 'bigwig', 'cool', 'gff', 'gff3', 'hic', 'maf', 'mcool', 'scool', 'vcf']
+            --trackmeta '$autoCollection[$key],$autoCollection[$key].ext,$key'
+        #else if $autoCollection[$key].ext in ['bam',]
+            --trackmeta '$autoCollection[$key],$autoCollection[$key].ext,$key,$autoCollection[$key].metadata.bam_index'
+        #else if $autoCollection[$key].ext in ['cram',]
+            --trackmeta '$autoCollection[$key],$autoCollection[$key].ext,$key,$autoCollection[$key].metadata.cram_index'
+        #end if
+    #end for
+    --outdir '$output.files_path'
+    --jbrowse2path \${JBROWSE2_PATH}
+    --sessName "Autogen JBrowse" &&
+    #if $jbgen.zipOut == "true":
+        (cd '$output.files_path' && zip -r - . ) > '$output'
+    #else
+      cp '$output.files_path/index.html' '$output'
+    #end if
+#else:
+    python '$__tool_directory__/jbrowse2.py'
```

Comment:
do we need to pass in here `--version` to clone the correct version of jbrowse?

Reply:
Not cloning here - the first #if is for a collection - the one starting in line 48 is for a normal form-driven jbrowse2 script run so no, not needed here I think?